test/Microsoft.AspNetCore.Razor.Language.Test/Legacy/BlockTest.cs

```diff
@@ -60,8 +60,17 @@ public void Clone_ClonesBlock()
             var copy = (Block)block.Clone();
 
             // Assert
-            ParserTestBase.EvaluateParseTree(copy, block);
             Assert.NotSame(block, copy);
+
+            if (GenerateBaselines)
+            {
+                // We want to write the original to the baseline and test it against the copy.
+                BaselineTest(block, verifySyntaxTree: false);
+            }
+            else
+            {
+                BaselineTest(copy, verifySyntaxTree: false);
+            }
```

Comment:
I am just going to revert these. These had to change because I wanted to remove all the manual syntaxtree evaluation logic in ParserTestBase but they are also being used in a lot of our partial parsing tests which is what is failing the CI builds. So, I'm going to undo that deletion.